eva-server/src/test/java/uk/ac/ebi/eva/server/repository/VariantEntityRepositoryTest.java

```diff
@@ -124,14 +130,72 @@ public void testVariantRegionIsFoundMultiple(){
 
     @Test
     public void testNonExistentVariantRegionIsNotFound(){
-        String chr = "20";
+        String chr = "11";
         int start = 61098;
         int end = 60916;
-        final List<VariantEntity> variantEntityList = variantEntityRepository.findByChrAndStartWithMarginAndEndWithMargin(chr, start, end);
+        List<VariantEntity> variantEntityList =
+                variantEntityRepository.findByRegionAndComplexFilters(chr, start, end, new ArrayList<>(), new ArrayList<>(), null, null,
+                                                                      null, null, null, null, new PageRequest(0, 1000000));
         assertNotNull(variantEntityList);
         assertTrue(variantEntityList.size() == 0);
     }
 
+    @Test
+    public void testRegionIsFoundWithConsequenceType() {
+        List<String> cts = new ArrayList<>();
+        cts.add("SO:0001627");
+        testFiltersHelper("11", 188000, 190000, new ArrayList<>(), cts, null, null, null, null, null, null, 94);
+    }
+
+    @Test
+    public void testRegionIsFoundWithMafGt() {
+        testFiltersHelper("11", 185000, 190000, new ArrayList<>(), new ArrayList<>(), VariantEntityRepository.RelationalOperator.GT, 0.125,
+                          null, null, null, null, 37);
+    }
+
+    @Test
+    public void testRegionIsFoundWithMafGtE() {
```

Comment:
Keep naming consistent with the other test class, which uses `Gte` instead of `GtE`. Comment about using `GreaterThanEquals` also applies here.